prysm/helm/templates/_volume_mounts.tpl

```diff
@@ -0,0 +1,8 @@
+{{- define "prysm.volumeMounts" }}
+{{- $uniqueVolumes := fromYaml (include "unique.volumes" . ) }}
+{{- range $volumePath, $volumeName := $uniqueVolumes}}
+- name: {{ $volumeName }}
+  mountPath: {{ $volumePath }}
+  readOnly: {{ ne $volumePath $.dataVolumePath }}
```

Comment:
```suggestion
  readOnly: {{ ne $volumePath .dataVolumePath }}
```

Reply:
didn't update this as `$` is needed to access the value in the `range` scope